Source/WebCore/platform/SharedBuffer.cpp

```diff
@@ -121,6 +150,27 @@ Ref<SharedBuffer> FragmentedSharedBuffer::makeContiguous() const
     return SharedBuffer::create(WTFMove(combinedData));
 }
 
+#if PLATFORM(COCOA)
+auto FragmentedSharedBuffer::toIPCData() const -> FragmentedSharedBuffer::FragmentedSharedBufferData
+{
+    size_t bufferSize = size();
+    if (!bufferSize)
+        return std::span<const uint8_t> { };
+
+    if (bufferSize < minimumPageSize) {
```

Comment:
Isn't it as easy as adding a `useUnixDomainSockets() ||` here to support non-COCOA ports now?

Reply:
yes, I can add that back